### PR TITLE
javascript2-week1/Sevda

### PR DESCRIPTION
**Pull request by Sevda-NZ**
Please review. Thanks in advance :)

### CODE REVIEW

File: 3_javascript/javascript2/week1/script.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+// Find and count the Danish letters
+function danishLetters(vowels) {
+  let obj = {};
```
Comment (szoboszlaianna) [inline 3_javascript/javascript2/week1/script.js:3]: the function could be made more efficient for example by using an object to keep track of the numbers:   ` const danishVowels = { "æ": 0, "ø": 0, "å": 0 }`.  Also would be nice to check for capital letters as well.

File: 3_javascript/javascript2/week1/script.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+button.addEventListener("click", () => {
+  let inputValue = input.value;
+  if (inputValue != "") {
+    let spiritName = `${inputValue} - The ${adjectives[randomNumber()]} ${
+      animals[randomNumber()]
+    }`;
+    let paragraph = document.createElement("p");
+    paragraph.innerText = spiritName;
+    document.querySelector("body").append(paragraph);
```
Comment (szoboszlaianna) [inline 3_javascript/javascript2/week1/script.js:75]: would be nice to create the previous one when generating a new.

File: 3_javascript/javascript2/week1/script.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+// Find and count the Danish letters
```
Comment (szoboszlaianna) [inline 3_javascript/javascript2/week1/script.js:1]: Where is the 3rd task?

**Review by szoboszlaianna — CHANGES_REQUESTED** [review]

**Comment by Sevda-NZ** [comment]
Reviews are implemented :)

**Pull request closed without merging**